bitsandbytes/cextension.py

```diff
@@ -70,6 +70,11 @@ def __init__(self, lib: ct.CDLL):
 
 
 def get_native_library() -> BNBNativeLibrary:
+    from bitsandbytes.backends.cpu_xpu_common import ipex_cpu, ipex_xpu
+
+    if ipex_cpu or ipex_xpu:
+        return None
```

Comment:
The type annotation for this function does not indicate it could ever return None.

Reply:
done

Reply:
What was done?

Reply:
Sorry for I forgot to remove `return None` in this function. Now, it's okay.